src/main/java/org/opensearch/knn/quantization/models/quantizationParams/ScalarQuantizationParams.java

```diff
@@ -25,6 +26,11 @@
 @EqualsAndHashCode
 public class ScalarQuantizationParams implements QuantizationParams {
     private ScalarQuantizationType sqType;
+    private boolean isEnableRandomRotation = QFrameBitEncoder.DEFAULT_ENABLE_RANDOM_ROTATION;
```

Comment:
nit: final? Also, do we use this somewhere?

Reply:
Good catch, I moved the `isEnableRandomRotation` to training request and didn't delete. I don't think scalarquantizationparams needs to store this and we can keep solely in training request and in the quantization state. Will remove.

Reply:
Took another look and `isEnableRandomRotation` is necessary for `QuantizationService::train` when that method constructs the training request.

If I make this field final I'll have to drop the `@AllArgsConstructor` and force the `@NoArgsConstructor`. What do you think?